operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -708,3 +708,31 @@ groups:
     for: 15m
     labels:
       severity: warning
+- name: mimir_continuous_test
+  rules:
+  - alert: MimirContinuousTestNotRunning
+    annotations:
+      message: Mimir continuous test {{ $labels.test }} in {{ $labels.cluster }}/{{
+        $labels.namespace }} is not effectively running because writes are failing.
+    expr: |
+      sum by(cluster, namespace, test) (rate(mimir_continuous_test_writes_failed_total[5m])) > 0
+    for: 1h
+    labels:
+      severity: warning
+  - alert: MimirContinuousTestNotRunning
```

Comment:
I'd use a different alert name for writes and queries. Message isn't used for grouping, so both of these would fall into the same group.

Reply:
Good point. Will fix.